src/main/java/com/alrex/parcool/client/input/KeyRecorder.java

```diff
@@ -29,6 +34,15 @@ public class KeyRecorder {
 	public static void onClientTick(TickEvent.ClientTickEvent event) {
 		if (event.phase != TickEvent.Phase.START) return;
 
+		if (KeyBindings.getKeyBindEnable().consumeClick()) {
```

Comment:
I think only recording of key status should be here and actual processing should be at new event consumer like these  ([com.alrex.parcool.common.event](https://github.com/alRex-U/ParCool/tree/main/src/main/java/com/alrex/parcool/common/event)). How do you think?

Reply:
Migrated. tested & works fine too.